packages/lesswrong/server/givingSeason/resolvers.ts

```diff
@@ -133,7 +166,6 @@ defineFeedResolver<number>({
             baseScore: {$gte: 5},
```

Comment:
We probably want to filter out deleted etc as well

Reply:
The results of `viewBasedSubquery` get run through `accessFilterMultiple`, so the default view will be applied automatically.